### ISSUE TITLE
FR: In template fieldtype allow filtering of available templates

### ISSUE THREAD

**Issue by iboxifoo** [statamic/v2-hub#1470]
Currently the template fieldtype lists all available templates in its dropdown. Would it be possible to allow filtering like in the Pages or Collection fieldtype, so users can only choose available templates from a configured list.

**Comment by jasonvarga** [comment]
In the mean time, you could just use a `suggest` or `select` fieldtype and hardcode the template names as options.

**Comment by iboxifoo** [comment]
@jasonvarga thanks for the tip. i thought there'd be a way around this. i can use such a variable in the front matter for an entry/page?

**Comment by jasonvarga** [comment]
Absolutely. Specifying a template would go into the front-matter anyway. The fieldtype is just how it's presented on the CP side.
